src/r2retdec.h

```diff
@@ -0,0 +1,9 @@
+#ifndef R2RETDEC_H
+#define R2RETDEC_H
+
+#include <r_util/r_annotated_code.h>
+#include <r_core.h>
+
+RAnnotatedCode* r2retdec_decompile_annotated_code(RCore *core, ut64 addr);
+
+#endif //R2RETDEC_H
```

Comment:
Can you move this file to the `include/r2plugin`?

With this update of `cutter-plugin/CMakeLists.txt` it should be required to add `core_retdec` [here]( https://github.com/avast/retdec-r2plugin/blob/1ba22472bf7718cf5b70130d249c95681ad71889/cutter-plugin/CMakeLists.txt#L18)

Reply:
Hi, I'm thinking of making a new PR from a new branch made from the current master of retdec-r2plugin. I tried rebasing it, but couldn't succeed with minimal effort. I think to make a PR from the new branch will be less time-consuming. Are you okay with this? Also, can you please tell me what exactly I should add in `cutter-plugin/CMakeLists.txt`.